lib/testLauncher/SuitestLauncher.js

```diff
@@ -26,64 +28,71 @@ class SuitestLauncher {
 		try {
```

Comment:
do we really need this try-catch? Almost all covered code will be executed inside child processes.

Reply:
There are also calls to server and we have to show to user a meaningful message in that case.
Second try/catch is only for "finally" part - to make sure session is closed on server properly if something goes wrong.

Reply:
what if closeSessionUnchained on 42 line will failed? We will get finally block and there closeSessionUnchained will be called twice.

Reply:
Sorry, I was a little mistaken, I talked about try catch on 37 line

Reply:
You're right, should have put second try somewhere after line 45. Will update.